content/en/docs/reference/kubectl/conventions.md

```diff
@@ -52,43 +52,6 @@ You can generate the following resources with a kubectl command, `kubectl create
   serviceaccount      Create a service account with the specified name.
 ```
 
-
-You can create the following resources using `kubectl run` with the `--generator` flag:
-
-{{< table caption="Resources you can create using kubectl run" >}}
-| Resource                             | API group          | kubectl command                                   |
-|--------------------------------------|--------------------|---------------------------------------------------|
-| Pod                                  | v1                 | `kubectl run --generator=run-pod/v1`              |
-| ReplicationController _(deprecated)_ | v1                 | `kubectl run --generator=run/v1`                  |
-| Deployment _(deprecated)_            | extensions/v1beta1 | `kubectl run --generator=deployment/v1beta1`      |
-| Deployment _(deprecated)_            | apps/v1beta1       | `kubectl run --generator=deployment/apps.v1beta1` |
-| Job _(deprecated)_                   | batch/v1           | `kubectl run --generator=job/v1`                  |
-| CronJob _(deprecated)_               | batch/v2alpha1     | `kubectl run --generator=cronjob/v2alpha1`        |
-| CronJob _(deprecated)_               | batch/v1beta1      | `kubectl run --generator=cronjob/v1beta1`         |
-{{< /table >}}
-
-{{< note >}}
-Generators other than `run-pod/v1` are deprecated.
-{{< /note >}}
```

Comment:
This note looks important. Is it still true?

Reply:
I'd say not supported anymore. They seem to be removed.
Please check following snippet
```
k run --restart=OnFailure --dry-run=client --image=busybox foo -oyaml
apiVersion: v1
kind: Pod <-------------- Here
metadata:
  creationTimestamp: null
  labels:
    run: foo
  name: foo
spec:
  containers:
  - image: busybox
    name: foo
    resources: {}
  dnsPolicy: ClusterFirst
  restartPolicy: OnFailure
status: {}
```

Reply:
So, the situation is that all generators are deprecated in v1.18 _but_ they are still part of Kubernetes. I think if someone sees a mention of a “generator” and looks it up, it's helpful if they find Kubernetes documentation that tells them about the concept and that it's something they shouldn't use.

Otherwise, what people will find is older third-party documentation that probably doesn't mention deprecation.

Reply:
It's important for docs to reflect current reality. Can we remove mention of generators _and_ add a note that they're deprecated as of 1.18? @n0npax Would you be willing to refactor this PR to include a 1.18 deprecation note?

Reply:
@jimangel Would a deprecation note that links to generators in 1.17 be helpful?

Reply:
I think that would be a great compromise.

Reply:
requested information added to line 35: https://github.com/kubernetes/website/pull/20107/files#diff-b9077c26b2045388bf1477e92fb799d8R35